net/cifsd-tools/Makefile

```diff
@@ -0,0 +1,59 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=cifsd-tools
+PKG_SOURCE_DATE:=2019-07-05
+PKG_SOURCE_VERSION:=539fa21a8dd427a8ca2dc13c9a5a1c975be96d3c
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_SOURCE_VERSION).tar.gz
```

Comment:
Not sure it matters, but PKG_SOURCE_VERSION could be replaced by PKG_SOURCE_DATE so that it can be sorted in order.

Alternatively, PKG_SOURCE could be eliminated by switching to local tarballs.

Reply:
I switched to git, i don't see the point of codeload on such a small repo and it still always confuses me.